c/tskit/tables.c

```diff
@@ -10460,6 +10659,22 @@ tsk_table_collection_write_format_data(const tsk_table_collection_t *self,
     return ret;
 }
 
+static int TSK_WARN_UNUSED
+tsk_reference_sequence_dump(const tsk_reference_sequence_t *self, kastore_t *store,
```

Comment:
I'd make this a method of the table collection as well, keep consistency with the nearby methods. Doesn't make sense to have a load method and no dump method, I think?

Reply:
Done